_targets.R

```diff
@@ -1,16 +1,20 @@
 
 library(targets)
+## why place these two packages here. vs in the tar_option_set()
```

Comment:
see my comment.
When asked to add `tarchetypes` and `tibble` by the bot, I initially put in under line 5, within the list of packages in `tar_option_set()`. However I could not run `tar_map()` as it was not a recognized R function. Are we supposed to load `tarchetypes` in the `_targets.R` the same way as targets package is loaded? What about the `tibble` package? Is this because these packages are directly relevant for the `_targets.R` script, which the packages in `tar_option_set()` are required for the other sourced scripts?